awswrangler/opensearch/_write.py

```diff
@@ -504,6 +504,7 @@ def index_documents(
     initial_backoff: int | None = None,
     max_backoff: int | None = None,
     use_threads: bool | int = False,
+    modify_refresh_interval: bool = True,
```

Comment:
Looks good, but perhaps `enable_refresh_interval` is a better name?

Reply:
sure... whatever works for you...